app/services/clients/publishing_api.rb

```diff
@@ -26,8 +26,18 @@ def fetch_all(fields)
         .flatten
     end
 
-    def fetch(content_id, locale)
-      normalise(publishing_api.get_content(content_id, locale: locale))
+    def fetch(content_id, options = {})
+      normalise(publishing_api.get_content(content_id, options))
+    end
+
+    def fetch_latest_published(content_id, locale)
```

Comment:
I think `fetch_latest_published` should be implicit within the `fetch` method. Audit Tool and Content Performance Manager only care about `published` content items.
It seems that the issue is related with having a draft version of a `ContentItem` created after a Content Item has been published. 
I am not feeling comfortable returning `nil` to express that the content item is drafted y their latest edition.
I would prefer to keep the `fetch` method as it is, and inside it, get the latest published version.

Reply:
Would you know as well, if we can have more than one draft version for a Content Item? If this is the case, the way it is implemented would not work in all situations.

Reply:
Fine by me. I kept `fetch`, because we still need to fetch draft / published stuff. But I can hide that implementation.

You can only have [one draft](https://github.com/alphagov/publishing-api/blob/master/doc/model.md#workflow):

> At any one time a document can contain:
>
> - 1 edition in a "draft" state
> - 1 edition in a "published" or "unpublished" state
> - any number of editions in a "superseded" state